src/components/organisms/Header/Header.tsx

```diff
@@ -257,4 +274,20 @@ const PointsTag = ({
     );
 };
 
+const StablecoinExternalLink = () => {
+    return (
+        <Link
+            href={getStablecoinAppUrl()}
+            target='_blank'
+            className={clsx(
+                'typography-mobile-body-5 tablet:typography-desktop-body-4 flex h-8 flex-shrink-0 items-center justify-center gap-1.5 rounded-lg bg-neutral-800 px-2.5 py-2.5 font-semibold text-neutral-50 ring-1 ring-neutral-500 active:border-transparent tablet:h-10 tablet:rounded-xl tablet:pr-3 tablet:ring-[1.5px]'
```

Comment:
would be good to have some hover styles here for border and/or background.

Reply:
Adding this